src/Dapr.Actors/Communication/ActorInvokeExceptionData.cs

```diff
@@ -0,0 +1,52 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Actors
+{
+    using System;
+    using System.IO;
+    using System.Runtime.Serialization;
+    using System.Xml;
+    using Dapr.Actors;
+    using Microsoft.Extensions.Logging;
+
+    [DataContract(Name = "ActorInvokeExceptionData", Namespace = Constants.Namespace)]
+    internal class ActorInvokeExceptionData
+    {
+        private static readonly DataContractSerializer ActorInvokeExceptionDataSerializer = new DataContractSerializer(typeof(ActorInvokeExceptionData));
+
+        public ActorInvokeExceptionData(string type, string message)
+        {
+            this.Type = type;
+            this.Message = message;
+        }
+
+        [DataMember]
+        public string Type { get; private set; }
+
+        [DataMember]
+        public string Message { get; private set; }
+
+        internal static ActorInvokeExceptionData Deserialize(Stream buffer)
```

Comment:
I'd suggest some naming consistency for these parameters. `Stream stream` or `byte[] buffer`. It's a little bit confusing to see `Stream buffer` - a `Stream` represents a read/writeable thing with no guarantees about where the data is - whereas a `byte[]` is actual buffered memory.